src/selectors/per-thread/composed.js

```diff
@@ -43,9 +43,16 @@ export function getComposedSelectorsPerThread(
    * when it's absurd.
    */
   const getUsefulTabs: Selector<$ReadOnlyArray<TabSlug>> = createSelector(
+    threadSelectors.getThread,
     threadSelectors.getIsNetworkChartEmptyInFullRange,
     threadSelectors.getJsTracerTable,
-    (isNetworkChartEmpty, jsTracerTable) => {
+    ({ processType }, isNetworkChartEmpty, jsTracerTable) => {
+      if (processType === 'comparison') {
+        // For a diffing tracks, we display only the calltree tab for now, because
+        // other views make no or not much sense.
+        return ['calltree'];
```

Comment:
Found a CSS visual inconsistency that I think is worth cleaning up (if it's easy).

When there is only one tab, then the the tab bar is 1px smaller, causing the layout to shift a bit. I think it's due to the background tabs being 1px taller. It would be nicer to make that 1px jump not happen.

Steps to reproduce:
 * Click on a normal thread
 * Be on the call tree
 * Click on a diff thread

Expected result:
 * The call tree doesn't visually shift

Actual result:
 * The call tree shifts up 1px.

Reply:
I think I found the reason, fixed it and more :)